TYPO3.Neos/Resources/Public/JavaScript/Content/Inspector/Editors/ImageEditor.js

```diff
@@ -1059,6 +1070,13 @@ function (Ember, $, FileUpload, template, cropTemplate, BooleanEditor, Spinner,
 		}.property('features.resize', '_originalImageUri'),
 
 		/**
+		 * Swaps the small preview inspector image with the original for cropping preview
+		 */
+		_swapPreviewWithOriginal: function() {
+				this.set('_inspectorImageUri', this.get('_originalImageUri'));
```

Comment:
In the end it's still the URI for the inspector image, even though we share the same preset as the thumbnail images in the media browser. I think the variable name is correct. There is already a "previewImageUri" and I think "_thumbnailImageUri" is not specific enough